rspace/src/main/scala/coop/rchain/rspace/serializers/ScodecSerialize.scala

```diff
@@ -166,41 +166,20 @@ object ScodecSerialize {
     (codecConsume :: codecSeq(codecProduce) :: sortedSet(uint8) :: codecMap(codecProduce, int32))
       .as[COMM]
 
-  val codecSkip: Codec[Skip] = (codecByteVector :: codecTrieValuePointer).as[Skip]
-
-  val memoizingSkipCodec: Codec[Skip] =
-    Codec.apply((s: Skip) => Attempt.successful(s.encoded), codecSkip.decode)
```

Comment:
Remove this because it cause infinite loop on calling method which full the stack

Reply:
Skip and PointerBlock are not abstract but concrete here.

Reply:
:+1: